frontend/rolecall/src/app/user/user-editor.component.ts

```diff
@@ -18,27 +17,24 @@ import { User, UserApi } from '../api/user_api.service';
 export class UserEditor implements OnInit {
 
   permissionsSet: EventEmitter<string[]> = new EventEmitter();
-  privilegeClassSet: EventEmitter<string[]> = new EventEmitter();
+  rolesSet: EventEmitter<string[]> = new EventEmitter();
 
   currentSelectedUser: User;
   renderingUsers: User[];
+  su: User[]; // Sorted Users
```

Comment:
Please call this "sortedUsers" to keep var names readable.

Reply:
When I went to fix this, I realized that the variable wasn't used. (I was only used for a debug printout earlier.) This inspired me to try to lint the project.
Tons of problems: missing semicolons, let that should be const, etc. etc.
I am tempted to do a separate PR for fixes to the lint errors.

Reply:
Yeah we'll fix the lints and nits as we go :)